src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/DocumentWithDescription.java

```diff
@@ -17,16 +18,19 @@ public DocumentWithDescription(
         Document document,
         String description
     ) {
-        this.document = document;
-        this.description = description;
+        requireNonNull(document);
+        requireNonNull(description);
+
+        this.document = Optional.ofNullable(document);
+        this.description = Optional.ofNullable(description);
```

Comment:
Are the `.ofNullable` required when we've run them through `requireNonNull()`?

Reply:
Shameful display! Not they are not required.